DANS-KNAW/easy-solr4files-index
# EASY-1320 custom vagrant without stripped versions of bags from act-easy

Fixes EASY-1230

#### When applied it will

* have a local VM in the project
* [x] make the solr schema (duplicated from dtap) avaialble for resue in dtap
* [x] not use too real bags for test data

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

In the project root
* `mvn clean install`
* `vagrant up`
* `curl http://test.dans.knaw.nl:8983/solr/fileitems/query\?q\=%2A%3A%2A`
  it should find nothing
* `vagrant ssh`
* `easy-update-solr4files-index init`
  it should report it indexed one bag for the pdbs store and nothing for the other stores
* `exit`
* The curl command above should now find 9 files,
  to reduce output extend the query with `\&fl=id\&rows\=1`
  or execute the default query with the browser on
  `http://test.dans.knaw.nl:8983/solr/#/fileitems/query`
* another `vagrant provision` should skip copying test bags as it take a while

The dev-tool scripts still run with deasy.


#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
dans-                    | ...  | ...
